crates/api/src/op_store.rs

```diff
@@ -0,0 +1,86 @@
+//! Kitsune2 op store types.
+
+use crate::{K2Result, OpId, Timestamp};
+use futures::future::BoxFuture;
+use std::cmp::Ordering;
+use std::sync::Arc;
+
+/// An op with metadata.
+///
+/// This is the basic unit of data in the kitsune2 system.
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct MetaOp {
+    /// The id of the op.
+    pub op_id: OpId,
+
+    /// The creation timestamp of the op.
+    ///
+    /// This must be the same for everyone who sees this op.
+    ///
+    /// The host must reject the op if the timestamp does not agree with any timestamps inside the
+    /// op data.
+    pub timestamp: Timestamp,
+
+    /// The actual op data.
+    pub op_data: Vec<u8>,
+
+    /// Crdt-style add-only opaque implementor-use flags.
```

Comment:
Should we give some more context about this? I don't understand it from just this comment

Reply:
Good question, this was copied from the original kitsune2-api spike. I think we actually agreed to make this a u32 and expose that as an enum to that humans don't have to look at it.

I think I should just remove this from the current PR and we'll add it in when we need op flags.